toolchains/BUCK

```diff
@@ -0,0 +1,6 @@
+load("@prelude//toolchains:genrule.bzl", "system_genrule_toolchain")
+
+system_genrule_toolchain(
```

Comment:
we will not need this, we will need to setup python toolchain [like this](https://github.com/facebook/buck2/blob/95dd7908111ed529637a321b0a365573e6b5cc30/prelude/toolchains/demo.bzl#L40-L48)